pages/desktop/consumer_pages/base.py

```diff
@@ -84,6 +86,29 @@ class HeaderRegion(Page):
         def is_user_logged_in(self):
             return self.is_element_visible(*self._settings_toggle_locator)
 
+        @property
+        def categories_name(self):
+            return self.selenium.find_element(*self._categories_toggle_locator).text
+
+        def open_categories_menu(self):
+            categories = self.selenium.find_element(By.ID, 'header--categories')
```

Comment:
I think this was conscious, but I can't recall my reasons, and it breaks our convention so I'll fix it.